MAUI/DataGrid/sorting.md

```diff
@@ -0,0 +1,252 @@
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.ShowSortNumbers = true;
+{% endhighlight %}
+{% endtabs %}
+
+![Sort with sequence numbers .NET MAUI DataGrid](Images\sorting\maui-datagrid-show-sort-numbers.png)
+
+## Sort column in double tap
+
+By default, column gets sorted when column header is clicked. This behavior can be changed to sort the column in double click action by setting [SfDataGrid.SortingGestureType](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SortingGestureType) property to [DoubleTap](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSortingGestureType.html#Syncfusion_Maui_DataGrid_DataGridSortingGestureType_DoubleTap).
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.SortingGestureType = DataGridSortingGestureType.DoubleTap;
+{% endhighlight %}
+{% endtabs %}
+
+## Sorting events
+
+The data grid provides the following events for the sorting functionality,
+
+* [SortColumnsChanging](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SelectionChanging): This event is invoked while sorting the column at execution time before the column gets sorted. It helps to cancel the sorting action by setting the `Cancel` property of [DataGridSortColumnsChangingEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSortColumnsChangingEventArgs.html).
+* [SortColumnsChanged](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SelectionChanged): This event is invoked after the column is sorted.
+These two events are triggered with `DataGridSortColumnsChangingEventArgs` and [DataGridSortColumnsChangedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSortColumnsChangedEventArgs.html) that contains the following properties:
+
+**AddedItems**: Gets the collection of `SortColumnDescription` objects that are added to `SortColumnDescriptions` collection for sorting.
+**RemovedItems**: Gets the collection of `SortColumnDescription` objects that are removed from `SortColumnDescriptions` collection.
+
+Following code snippet demonstrates how to cancel sorting for a particular column by using `SortColumnsChanging` event,
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid" SortingMode="Single" SortColumnsChanging="sfDataGrid_SortColumnsChanging" ItemsSource="{Binding OrderInfoCollection}" />
+
+private void sfDataGrid_SortColumnsChanging(object sender, DataGridSortColumnsChangingEventArgs e)

```

Comment:
@ashok-kuvaraja  - Why this code given inside the xaml? Seems something wrong. Can you revisit this?

Reply:
We have shown the same code changes in both XAML and c#. So, If I invoke any event inside the XAML code, I need to maintain it inside the XAML section. We can't show it separately by a new tap. We approached the same way that what we applied in Xamarin.